### PR TITLE
fix React dom errors in `ButtonAssets` as well as warnings in some stories

### PR DESCRIPTION
**Pull request by nedredmond**
Fixes invalid SVG properties in `ButtonAssets.tsx`-- switches from kebab-case to camelCase.

Also adjusts story args to eliminate warnings.

### CODE REVIEW

File: .changeset/strange-pillows-argue.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/math-input": patch
+---
+
+Fix dom errors from SVG markup and adjust stories to remove warnings
```
Comment (jeremywiebe) [inline .changeset/strange-pillows-argue.md:5]: **thought**: I don't think we need a changeset for this change.  My reasoning for whether I need to do a Changeset is if the change affects consumers of this package. I guess the DOM errors could be relevant to consumers, but not the stories change.

**Review by jeremywiebe — APPROVED** [review]
Thanks for fixing those SVG errors!

File: .changeset/strange-pillows-argue.md (hunk 1 above)
Comment (nedredmond) [reply .changeset/strange-pillows-argue.md:5]: Isn't that what patches are for? Perseus CI yells at me if there is no changeset

Comment (nedredmond) [reply .changeset/strange-pillows-argue.md:5]: how I understand it:
Path = code change with no apparent change for user
Minor = additional, compatible features that do not affect current consumers
Major = non-compatible changes to API that need attention from current consumers

Comment (nedredmond) [reply .changeset/strange-pillows-argue.md:5]: how do we interpret semver in this codebase?

Comment (jeremywiebe) [reply .changeset/strange-pillows-argue.md:5]: Your understanding of semver is correct and we use that interpretation for Perseus as well.  But the difference I'm pointing out is that changsets are a way to communicate with world outside of this package (they form the basis of the changelogs that we publish). For internal changes, I think the more standard way to see what changed would be to just look at the PRs included in a release. 

You can always use `yarn changeset --empty` to create a changeset file that is empty that will satisfy the Changeset Github Action.

Comment (nedredmond) [reply .changeset/strange-pillows-argue.md:5]: Oh, gotcha! Noted.

**Pull request merged**